src/import_vdf.py

```diff
@@ -31,16 +33,45 @@ def import_milvus(args):
     milvus_import = ImportMilvus(args)
     milvus_import.upsert_data()
 
+
 def import_qdrant(args):
     """
     Import data to Qdrant
     """
     set_arg_from_input(
         args,
         "url",
-        "Enter the url of Qdrant instance (default: 'http://localhost:6333'): ",
+        "Enter the URL of Qdrant instance (default: 'http://localhost:6334'): ",
         str,
-        "http://localhost:6333",
+        "http://localhost:6334",
+    )
+    set_arg_from_input(
+        args,
+        "prefer_grpc",
+        "Whether to use GRPC. Recommended. (default: True): ",
+        bool,
+        True,
+    )
+    set_arg_from_input(
+        args,
+        "parallel",
+        "Enter the batch size for upserts (default: 64): ",
+        int,
+        1,
+    )
+    set_arg_from_input(
+        args,
+        "batch-size",
+        "Enter the number of parallel processes of upload (default: 1): ",
```

Comment:
I think these are switched around. Also, should the default for parallel be higher (to speed up upsert)?